apps/medical-illustration/src/app/app-web-component.ts

```diff
@@ -0,0 +1,119 @@
+import { HttpClient } from '@angular/common/http';
+import { ChangeDetectionStrategy, Component, EventEmitter, inject, Input, OnChanges, Output } from '@angular/core';
+import { NodeMapEntry } from '@hra-ui/components/molecules';
+import { BehaviorSubject, Observable, tap } from 'rxjs';
+
+import { CellEntry, IllustrationData, JsonLd } from './models';
+
+/**
+ * Medical illustration web component
+ */
+@Component({
+  selector: 'hra-root-wc',
+  templateUrl: 'app-web-component.html',
+  styleUrls: ['app-web-component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class AppWebComponent implements OnChanges {
+  /** Http client */
+  private readonly http = inject(HttpClient);
+
+  /** Ftu illustrations file: can be url or file object */
+  @Input() lookupSrc: string | JsonLd = '';
+
+  /** Illustration info: can be illustration id or illustration data object */
+  @Input() illustrationSrc: string | IllustrationData = '';
+
+  /** Ontology id to highlight */
+  @Input() highlightId = '';
+
+  /** Emits node data when node hovered */
+  @Output() readonly nodeHovered = new EventEmitter<NodeMapEntry>();
+
+  /** Emits node data when node clicked */
+  @Output() readonly nodeClicked = new EventEmitter<NodeMapEntry>();
+
+  /** Illustration svg url behavior subject */
+  readonly url$ = new BehaviorSubject<string>('');
+
+  /** Mapping data behavior subject */
+  readonly mapping$ = new BehaviorSubject<NodeMapEntry[]>([]);
+
+  /** Highlighted ontology id behavior subject */
+  readonly highlight$ = new BehaviorSubject<string>('');
+
+  /**
+   * Sets illustration url and mapping data on input changes
+   */
+  ngOnChanges() {
+    this.highlight$.next(this.highlightId);
+    if (typeof this.lookupSrc === 'string') {
+      this.getData(this.illustrationSrc, this.lookupSrc).subscribe();
+    } else {
+      // if lookupSrc is a json file
+      this.setUrlAndMapping(this.lookupSrc, this.illustrationSrc);
+    }
+  }
+
+  /**
+   * Gets illustration data and sets illustration url and mapping data
+   * @param illustrationSrc Illustration src
+   * @param lookupSrc Lookup src
+   * @returns Observable
+   */
+  private getData(illustrationSrc: IllustrationData | string, lookupSrc: string): Observable<JsonLd> {
+    return this.http.get<JsonLd>(lookupSrc).pipe(
```

Comment:
Don't you have to specify that the `.get(...)` should return json or is that the default?